engine-kommand/build.gradle.kts

```diff
@@ -0,0 +1,98 @@
+import com.saveourtool.processbuilder.configureDetekt
+import com.saveourtool.processbuilder.configureDiktat
+
+plugins {
+    kotlin("multiplatform")
+}
+
+kotlin {
+    jvm {
+        jvmToolchain(11)
```

Comment:
why 11?

Reply:
Will change to 8

Reply:
Actually won't as I use features from jvm 9